go.sum

```diff
@@ -33,8 +33,8 @@ github.com/TRON-US/go-btfs-chunker v0.2.7 h1:eLIZjETFbCsQ2lY9gw97x8MHZJkfOUfsZ05
 github.com/TRON-US/go-btfs-chunker v0.2.7/go.mod h1:Wj0oyybAWtu5lpcAc90QQ3bhJ14JRXD50PqxP25wmnI=
 github.com/TRON-US/go-btfs-cmds v0.1.5 h1:AbnAPBAFxe67MPChbwFPJbyKY1Vm3lXybGDNQ4NUTno=
 github.com/TRON-US/go-btfs-cmds v0.1.5/go.mod h1:rtki4vmPzq7qmjdzThJELFpSpxTiORoXreHlExdI6eg=
-github.com/TRON-US/go-btfs-config v0.3.2 h1:XIwdSQasv0gPkuiBKyrg2GRs24ksE+khLPzMvSZRzt4=
-github.com/TRON-US/go-btfs-config v0.3.2/go.mod h1:045+FmBpTIWNy34ISzjtPaEf6afBq6wVQtviNmAnF5g=
+github.com/TRON-US/go-btfs-config v0.4.0-beta3 h1:R5DXP6mYRSdo+RButh6sa6lvpDfFCRRIEVBPL7D8ng0=
```

Comment:
update to https://github.com/TRON-US/go-btfs-config/releases/tag/v0.4.0 instead?

Reply:
yeah let me create a new branch first. dont want to commit to release -> review -> master the chain is too long